src/components/weather-forecast.js

```diff
@@ -0,0 +1,28 @@
+const WeatherForecast = () => {
+  return (
+    <div className="forecast-container">
+      <div className="row">
+        <div className="col-md-3">
+          <h3>City</h3>
+        </div>
+      </div>
+      <div className="row">
+        <div className="col-md-3">
+          <h3>Temperature</h3>
```

Comment:
You probably want the headers to be separate from the Redux store data, since those are static